component/s3storage/s3storage_test.go

```diff
@@ -2087,6 +2087,16 @@ func (s *s3StorageTestSuite) TestReadDir() {
 
 func (s *s3StorageTestSuite) TestStreamDirSmallCountNoDuplicates() {
 	defer s.cleanupTest()
+
+	// due to LocalStack issue https://github.com/localstack/localstack/issues/9605
+	// LocalStack will list the subdirectory twice
+	// the only guess we have here for whether LocalStack is running is using the OS
+	//   since LocalStack only runs on Linux in our tests
+	if runtime.GOOS != "windows" {
```

Comment:
With the bug fix to local stack, do we need this, or can we just make sure we are using the latest docker version of localstack?

Reply:
Greate! I'll drop this!